### ISSUE TITLE
Finding more information on excluded variants

### ISSUE THREAD

**Issue by BoPeng** [vatlab/VariantTools#105]
We are currently using the Vtools platform to run some SKAT tests in an ongoing project. We have run the SKAT test and would further like to get more information regarding the variants that are included in the SKAT test. Which variants are behind the assoc results?
 
I have previously generated txt files with the 'output' option as in for example:
```
vtools output tablex variant.chr variant.pos refGene.name2 variant.aff_num ExAC.NFE_MAF > -t info_tablex.txt
```
 
However, I struggle to get a similar output after adding the options `--discard_samples "%(NA)>0.1"` and  `--discard_variants "%(NA)>0.1"'` in the SKAT test.

I'm interested to get a view of which variants that are finally included in the SKAT test. So, is it possible to make a table where the criteria `'--discard_samples "%(NA)>0.1"` and  `--discard_variants "%(NA)>0.1"` have been applied?

**Comment by BoPeng** [comment]
@gaow Any quick thought on this?

**Comment by gaow** [comment]
Sorry am traveling but a quick answer is that we can not make a sqlite table easily. But there is a test that simply dumps association data to the disk that can help? Cannot remember the command name now

**Comment by BoPeng** [comment]
The "dump data" association method is the `GroupWrite` method described [here](https://vatlab.github.io/vat-docs/applications/association/basic_data_statistics/group_stat_write/). It does not accept `NA` filtering though.

@jma7 How difficult it is to support the filtering parameters for `GroupWrite`?

**Comment by jma7** [comment]
So you want the ```GroupWrite``` command line looks like:
```
vtools associate rare status -m "GroupWrite test --discard_variants '%(NA)>0.1' " --group_by refGene.name2 --to_db gstat -j8
```
Then in the *_geno.txt file, only variants passed criteria will be saved?

**Comment by BoPeng** [comment]
Yes, this will allow the user to reproduce the behavior of other tests that use such filters.

**Comment by jma7** [comment]
It seems
```
vtools associate rare status -m "GroupWrite test" --discard_variants "%(NA)>0.001" --group_by refGene.name2 -f --to_db gstat -j8
```
will do the job?
Just put ```--discard_variants "%(NA)>0.001"``` in  ```vtools associate``` command line.
